guide/src/web-getting-started.md

```diff
@@ -60,100 +78,28 @@ ferrostar-core {
 }
 ```
 
-`<ferrostar-core>` web SDK contains an integrated search box, and you can already use it for navigation without any additional setup.
+That’s all you need to get started!
 
-## Configure the `<ferrostar-core>` component
+### Configuration explained
 
 `<ferrostar-core>` provides a few properties to configure.
 Here are the most important ones:
 
-- `httpClient`: You can set your own fetch-compatible HTTP client to make requests to the Valhalla endpoint.
-- `costingOptions`: You can set the costing options for the Ferrostar routing engine.
-- `useIntegratedSearchBox`: You can disable the integrated search box and use your own code to handle navigation.
-
-## Use your own code to handle navigation
-
-If you want to use your own code to handle navigation instead of the integrated search box, you can do so by the following steps:
-
-### (Optional) Implement your own search box
-
-You can use this code to retrieve the latitude and longitude of a destination:
-
-```javascript
-const destination = "One Apple Park Way";
-
-const { lat, lon } = await fetch("https://nominatim.openstreetmap.org/search?q=" + destination + "&format=json")
-  .then((response) => response.json())
-  .then((data) => data[0]);
-```
+- `valhallaEndpointUrl`: The Valhalla routing endpoint to use. You can use any reasonably up-to-date Valhalla server, including your own. See [vendors](./vendor.md#routing) for a list of known compatible vendors.
+- `httpClient`: You can set your own fetch-compatible HTTP client to make requests to the routing API (ex: Valhalla).
+- `costingOptions`: You can set the costing options for the route provider (ex: Valhalla JSON options).
+- `useIntegratedSearchBox`: Ferrostar web includes a search box powered by Stadia Maps, but you can disable this and replace with your own.
 
-### Configure the Ferrostar Core
-
-(TODO)
-Here's an example:
-
-```javascript
-const config = {
```

Comment:
this `config` object is required for `core.startNavigation`, but I'm not sure how to document it correctly. Also, should we include this as a default value for `core.startNavigation` in our web component code?

Reply:
> This config object is required for core.startNavigation

Actually we made a change very recently which moves the config requirement to the `FerrostarCore` constructor on other platforms ([Android example](https://github.com/stadiamaps/ferrostar/blob/2e6dfbd7e95a65e3fc18406e21f8d9c402be7077/android/core/src/main/java/com/stadiamaps/ferrostar/core/FerrostarCore.kt#L62)). I think doing the same on the web would be reasonable too. Most of the time you want to just set a config once. This makes it an _optional_ override when calling `startNavigation`.

> Also, should we include this as a default value for core.startNavigation in our web component code?

Yeah, the above addresses this, but it should be an optional override with a default set on the `FerrostarCore` component. Somewhat related: the `startNavigationFromSearch` internal function currently just uses hard-coded values; storing the config at the component level can fix this.

Re: documentation, I think it's good enough for now as the most common path is clear. We can improve customization docs as we refine the design.